lib/src/dio_http_formatter_base.dart

```diff
@@ -5,6 +5,8 @@ import 'package:logger/logger.dart';
 
 typedef HttpLoggerFilter = bool Function();
 
+const _startTimeKey = ' dio_http_formatter@start_time';
```

Comment:
Better to have `dio_http_formatter` as its own const variable that's prepended to each key we use.

So it would be: 

```dart
const _prefix = 'dio_http_formatter';
const _startTimeKey = '$_prefix.start_time';
```